distributions/standalone/assembly.xml

```diff
@@ -142,6 +142,7 @@
                 <include>org.mule.modules:mule-module-cxf</include>
                 <include>org.mule.modules:mule-module-extensions-support</include>
                 <include>org.mule.modules:mule-module-extensions-spring-support</include>
+                <include>org.mule.modules:mule-module-extensions-xml-support</include>
```

Comment:
I didn't. 
I'll fix the white list and run a -Pdistributions to see what's happening